api/diagnostics.go

```diff
@@ -507,6 +507,16 @@ func (j *DiagnosticsJob) getHTTPAddToZip(node Node, endpoints map[string]string,
 			j.JobProgressPercentage += percentPerURL
 			continue
 		}
+
+		if resp.StatusCode != http.StatusOK {
+			errMsg := fmt.Sprintf("unable to fetch %s. Return code %d", fullURL, resp.StatusCode)
+			logrus.Error(errMsg)
+			j.Errors = append(j.Errors, errMsg)
+			updateSummaryReport("", node, errMsg, summaryErrorsReport)
```

Comment:
Why do we use a blank string for the first argument to `updateSummaryReport` here?   The others seem to add the error message for this argument.

Reply:
fixed